working/0546-patterns/patterns-feature-specification.md

```diff
@@ -144,8 +144,8 @@ matching constructs to make working with that style enjoyable:
 ```dart
 double calculateArea(Shape shape) =>
   switch (shape) {
-    case Square(length: l) => l * l;
-    case Circle(radius: r) => math.pi * r * r;
+    case Square(length: var l) => l * l;
```

Comment:
🎉 this makes things way more clear imo

Reply:
I think so too, but I really appreciate the feedback. We went back and forth on this *so* much. :)